client/python/README.md

```diff
@@ -1,19 +1,73 @@
-# Armada Python client
+Armada Python client
+=
+
+# Installation:
+
+On Linux Machine
+=
+1) Open your favorite shell, for example, good old [Bourne Again SHell, aka, bash](https://www.gnu.org/software/bash/), the somewhat newer [Z shell, aka, zsh](https://www.zsh.org/), or shiny new [fish](https://fishshell.com/).
+2) Install [Git](https://git-scm.com/) by running `sudo apt-get install git-all` on [Debian-based](https://www.debian.org/) distributions like [Ubuntu](https://ubuntu.com/), or sudo dnf install git on [Fedora](https://getfedora.org/) 
+   and closely-related [RPM-Package-Manager](https://rpm.org/) based distributions like [CentOS](https://www.centos.org/). For further information see [Installing Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git).
+3) Fork the G-Research Armada repository to your personal repository.
```

Comment:
Forking is not required to install armada python client.

Additionally, our suggested install method should be the pypi package -- https://pypi.org/project/armada-client/ -- not a git-based version.